tools/files/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - stable-*
+    tags:
+      - "*"
+
+jobs:
+  linters:
+    uses: ./.github/workflows/linters.yml # use the callable linters job to run tests
+  ansible-lint:
+    uses: ansible/ansible-content-actions/.github/workflows/ansible_lint.yaml@main
+  changelog:
+    uses: ansible/ansible-content-actions/.github/workflows/changelog.yaml@main
+  sanity:
+    uses: ansible/ansible-content-actions/.github/workflows/sanity.yaml@main
+  units:
+    uses: ansible-network/github_actions/.github/workflows/unit_source.yml@main
+
+  all_green:
+    if: ${{ always() }}
+    needs:
+      - linters
+      - ansible-lint
+      - sanity
+      - units
+      - ansible-lint
+    runs-on: ubuntu-latest
+    steps:
+      - run: >-
+          python -c "assert set([
+          '${{ needs.changelog.result }}',
```

Comment:
We must also consider that changelog check might be skipped and if we do not handle this situation the assertion will fail even though it should not fail.

Reply:
It looks like instead of asserting `success` in the result, ansible-content-creator is doing `python -c "assert 'failure' not in set(...` which I think would handle the skipped changelog scenario.